test/e2e/devconsole.operatorsource.catalog.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: devconsole-operator-catalog
+  namespace: openshift-operators
```

Comment:
According to the [official example](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/catalogsourceconfig.cr.yaml):

* [`metadata.namespace`](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/catalogsourceconfig.cr.yaml#L5) should be `openshift-marketplace`